drone-fly-app/src/main/java/com/expediagroup/dataplatform/dronefly/app/service/listener/DefaultDroneFlyListener.java

```diff
@@ -0,0 +1,133 @@
+package com.expediagroup.dataplatform.dronefly.app.service.listener;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.metastore.MetaStoreEventListener;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.metastore.events.AddPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.AlterPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.AlterTableEvent;
+import org.apache.hadoop.hive.metastore.events.CreateDatabaseEvent;
+import org.apache.hadoop.hive.metastore.events.CreateFunctionEvent;
+import org.apache.hadoop.hive.metastore.events.CreateTableEvent;
+import org.apache.hadoop.hive.metastore.events.DropDatabaseEvent;
+import org.apache.hadoop.hive.metastore.events.DropFunctionEvent;
+import org.apache.hadoop.hive.metastore.events.DropPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.DropTableEvent;
+import org.apache.hadoop.hive.metastore.messaging.EventMessage.EventType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultDroneFlyListener extends MetaStoreEventListener {
+  private static final Logger log = LoggerFactory.getLogger(DefaultDroneFlyListener.class);
+
+  public DefaultDroneFlyListener(Configuration config) {
+    super(config);
+  }
+
+  @Override
+  public void onAddPartition(AddPartitionEvent addPartitionEvent) throws MetaException {
+    if (!addPartitionEvent.getStatus()) {
+      return;
+    }
+    log.info("Event Type: {}", EventType.ADD_PARTITION.toString());
```

Comment:
Can we change these to log everything on one line (using String.format() probably the nicest) and also log the status instead of returning above?

Reply:
Is this more readable ?

```
   log
        .info("Event Type: {}, DB Name: {}, Table Name: {}", EventType.ADD_PARTITION.toString(),
            addPartitionEvent.getTable().getDbName(), addPartitionEvent.getTable().getTableName());
```

Reply:
I think that will look better in the log file, so yes (and yeah, that's better than String.format, I forget slf4j can do this).

What are we going to do about the status? I'd like to print it but if it's failed then could we get NPE's on some of the other values? Perhaps we should just log the type and the status on line 30 before returning?

Reply:
I added status as well. I just learnt yesterday that Hive's CREATE_TABLE event actually sets `status = false` when you do "CREATE TABLE" on an already existing table. I am not sure when status is false other than that. But the event had all other values populated. So, there won't be null pointer I think.

Reply:
OK, cool.